qemu/tests/uefishell.py

```diff
@@ -200,6 +200,15 @@ def form_ping6_args():
                 params, "ipv6", True).split("%")[0]
         return " ".join([src_ipv6[0], target_ipv6])
 
+    def handle_memory_map(output):
+        """
+        compare command output with the output with pattern
+        """
+        rt_code = re.findall(params["check_rt_code"], output, re.M)
```

Comment:
according to the manual case, IMO, there are two assertions:
1. 800000000000000F for all RT_Code entries
2. The following never appears
RT_Code    000000007D6D5000-000000007D6F4FFF 0000000000000020 800000000000000F
RT_Code    000000007D6F5000-000000007D6F9FFF 0000000000000005 800000000002000F

So, what about checking the following:
1. search all RT_Code lines from output, each line ends with 800000000000000F(this can be assigned by an option in cfg)
2. search two adjacent RT_Code lines, the result should be None.

Reply:
agree with you. updated, thanks.